content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,207 @@
+# Application Registration Prerequisites / Developer Guide
+
+## Audience
+
+This technical specification is targeted at software developers building applications on top of Cerner's Ignite application platform as part of the [Cerner CODE ecosystem](https://code.cerner.com/).
+
+## Overview & Purpose
+
+This technical specification outlines a series of security best practices that, if followed, enable Cerner to provide more detailed information about your application to end users. The goal of these best practices is to encourage the use of existing internet security features and standards. In particular, we want users to:
+
+- be assured that their data is being securely transmitted using modern transport layer security and encryption protocols
+- understand what legal entity they're sharing their data with
+
+These best practices should be considered as strongly-recommended prerequisites that software developers address during the application registration process, prior to making their applications available for general use.  Failure to comply with these prerequisites will result in Cerner describing your application as an "Unknown Application" to users during the OAuth authorization workflow.
+
+### Glossary of Terms
+
+* eTLD - Effective Top Level Domain is the base domain. eTLD values are defined in the [Public Suffix List](https://publicsuffix.org/list/public_suffix_list.dat).  Examples include: .com, .edu, .org, and .co.uk.
+* eTLD+1 - "Effective top level domain plus 1" describes a domain name that is one level deeper than the effective top level domain (eTLD). Examples include examplehealth.com, cerner.com and cerner.co.uk.
+* OV - Organization Validated - a certificate issuance process that involves the certificate authority validating the organization (business entity) requesting the certificate.
+* EV - Extended Validation - a certificate issuance process that involves the certificate authority performing extended validation of the organization (business entity) requesting the certificate.
+* DV - Domain Validated - a certificate issuance process that involves the certificate authority validating the requester owns the domain. No validation of the business entity is performed.
+* TLS - Transport Layer Security
+* DNS TXT record - a "text" record in the Domain Name System (DNS).  TXT records usually take form of name=value.
+
+### Technical Requirements
+
+The following technical requirements are stated using RFC-2119 terminology. These requirements focus on the security best practices that Cerner expects application developers to adhere to in order for Cerner to provide the best description of your application to end users:
+
+* Web applications MUST support the HTTPS 1.1 protocol.
+* Application SHOULD support TLS.  (Please refer to the Cerner CODE Program's TLS guidance document for further detailed practices related to TLS.)
```

Comment:
Please add a link to the TLS guidance http://fhir.cerner.com/authorization/tls-guidance/

Reply:
+1 FWIW - There was a link to the TLS Guidance in the original draft:  https://wiki.cerner.com/pages/viewpage.action?pageId=2663941696

Reply:
I made changes to the files in the pull request and added a link.